### PR TITLE
AA-357: Add reset dates banner to outline tab

### PR DESCRIPTION
**Pull request by ciduarte**

### CODE REVIEW

File: src/courseware/course/course-exit/CourseCelebration.jsx [hunk 1]
```diff
@@ -189,18 +189,18 @@ function CourseCelebration({ intl }) {
...
                 >
                   {buttonText}
                 </Button>
               )}
               {buttonLocation && (
                 <Button
+                  className="bg-white"
                   variant="outline-primary"
                   href={buttonLocation}
-                  style={{ backgroundColor: 'white' }}
```
Comment (ciduarte) [inline src/courseware/course/course-exit/CourseCelebration.jsx:203]: unrelated change but realized I could switch this syntax

File: src/course-home/dates-banner/DatesBanner.jsx [hunk 2]
```diff
@@ -14,17 +15,17 @@ function DatesBanner(props) {
...
           {intl.formatMessage(messages[`datesBanner.${name}.body`])}
         </div>
         {bannerClickHandler && (
-          <div className="col-auto col-md-3 p-md-0 d-inline-flex align-items-center justify-content-start justify-content-md-center">
-            <button type="button" className="btn rounded align-self-center border border-primary bg-white mt-3 mt-md-0 font-weight-bold" onClick={bannerClickHandler}>
+          <div className="col-auto col-lg-3 p-lg-0 d-inline-flex align-items-center justify-content-start justify-content-lg-center">
+            <Button variant="outline-primary" className="align-self-center bg-white mt-3 mt-lg-0" onClick={bannerClickHandler}>
               {intl.formatMessage(messages[`datesBanner.${name}.button`])}
-            </button>
+            </Button>
```
Comment (ciduarte) [inline src/course-home/dates-banner/DatesBanner.jsx:28]: Now that this banner is on the Outline Tab, the rectangular button clashes with the rounded buttons we use in this view. I'm waiting for confirmation from Eugene that this change is okay.

(I made the styling change to show her the rounded button in action, but if she says to keep the rectangular button I can just remove these changes)

Comment (ciduarte) [reply src/course-home/dates-banner/DatesBanner.jsx:28]: This change is Eugene approved 👍

File: src/course-home/data/api.js [hunk 3]
```diff
@@ -145,9 +145,11 @@ export async function getOutlineTabData(courseId) {
   const courseGoals = camelCaseObject(data.course_goals);
   const courseExpiredHtml = data.course_expired_html;
   const courseTools = camelCaseObject(data.course_tools);
+  const datesBannerInfo = camelCaseObject(data.dates_banner_info);
   const datesWidget = camelCaseObject(data.dates_widget);
   const enrollAlert = camelCaseObject(data.enroll_alert);
   const handoutsHtml = data.handouts_html;
+  const hasEnded = camelCaseObject(data.has_ended);
```
Comment (Dillon-Dumesnil) [inline src/course-home/data/api.js]: This is just a boolean. I don't think we need the camelCaseObject on it.

**Review by Dillon-Dumesnil — APPROVED** [review]
One small change. LGTM

**Pull request merged**